samples/subsys/logging/logger/src/main.c

```diff
@@ -191,6 +191,32 @@ static void log_strdup_showcase(void)
 	transient_str[0] = '\0';
 }
 
+static void wait_on_log_flushed(void)
+{
+	while (log_buffered_cnt()) {
+		k_sleep(K_MSEC(5));
+	}
+}
+
+/* Wait for the moment when timestmap value changes and return current time
+ * stamp. In case of low frequency source we want to be at the edge.
+ */
+static u32_t timestamp_get_and_sync(void)
+{
+	u32_t start_timestamp = timestamp_get();
+
+	while (start_timestamp == timestamp_get()) {
+#if (CONFIG_ARCH_POSIX)
+		/* In the POSIX arch, time does not pass inside infinite loops.
+		 * Add a small delay in each iteration of those loops
+		 */
+		k_busy_wait(100);
+#endif
+	}
+
+	return timestamp_get();
```

Comment:
```suggestion
	return cur_timestamp;
```

The purpose of the proposed changes to these functions is to avoid another change in the timestamp on this line.

Reply:
thanks for suggestion, applied.